2024-03-26 15:56:06 INFO [32mpytest tests/backend/test_create_build_conf.py::TestBuildConfCreate::test_positive_creating_build_conf[By agent manager] (setup)[0m
curl - X GET 'http://localhost:8111/authenticationTest.html?csrf' 
-H 'User-Agent:python-requests/2.31.0'
-H 'Accept-Encoding:gzip, deflate'
-H 'Accept:*/*'
-H 'Connection:keep-alive'
-H 'Authorization:Basic OjIyMDgzMzE4MDU1NDMxNzM5NjQ=' 
-d ''
2024-03-26 15:56:06 INFO [32mpytest tests/backend/test_create_build_conf.py::TestBuildConfCreate::test_positive_creating_build_conf[By agent manager] [0m
curl - X POST 'http://localhost:8111/app/rest/users' 
-H 'User-Agent:python-requests/2.31.0'
-H 'Accept-Encoding:gzip, deflate'
-H 'Accept:application/json'
-H 'Connection:keep-alive'
-H 'Content-Type:application/json'
-H 'X-TC-CSRF-Token:ab464863-767f-4d39-af89-f539280c81fd'
-H 'Cookie:RecentLogin=; TCSESSIONID=B053D613E27F77C01F5142CDC9813C38'
-H 'Content-Length:149'
-H 'Authorization:Basic OjIyMDgzMzE4MDU1NDMxNzM5NjQ=' 
-d '{"username": "admit", "password": "n4FthiwCIom", "email": "parkertravis@example.net", "roles": {"role": [{"roleId": "AGENT_MANAGER", "scope": "g"}]}}' 
 

2024-03-26 15:56:06 INFO 	RESPONSE:
STATUS_CODE: [31m400[0m
DATA: [31mResponding with error, status code: 400 (Bad Request).
Details: jetbrains.buildServer.server.rest.errors.BadRequestException: Cannot create user as user with the same username already exists, caused by: jetbrains.buildServer.users.DuplicateUserAccountException: The specified username 'admit' is already in use by some other user.
Invalid request. Please check the request URL and data are correct.[0m
2024-03-26 15:56:06 INFO [32mpytest tests/backend/test_create_build_conf.py::TestBuildConfCreate::test_positive_creating_build_conf[By agent manager] (teardown)[0m
curl - X GET 'http://localhost:8111/authenticationTest.html?csrf' 
-H 'User-Agent:python-requests/2.31.0'
-H 'Accept-Encoding:gzip, deflate'
-H 'Accept:application/json'
-H 'Connection:keep-alive'
-H 'Content-Type:application/json'
-H 'X-TC-CSRF-Token:ab464863-767f-4d39-af89-f539280c81fd'
-H 'Cookie:TCSESSIONID=B053D613E27F77C01F5142CDC9813C38'
-H 'Authorization:Basic OjIyMDgzMzE4MDU1NDMxNzM5NjQ=' 
-d '' 
 

2024-03-26 15:56:06 INFO [32mpytest tests/backend/test_create_build_conf.py::TestBuildConfCreate::test_positive_creating_build_conf[By agent manager] (teardown)[0m
curl - X DELETE 'http://localhost:8111/app/rest/buildTypes/id:YVKCZRsoJht' 
-H 'User-Agent:python-requests/2.31.0'
-H 'Accept-Encoding:gzip, deflate'
-H 'Accept:application/json'
-H 'Connection:keep-alive'
-H 'Content-Type:application/json'
-H 'X-TC-CSRF-Token:ab464863-767f-4d39-af89-f539280c81fd'
-H 'Cookie:TCSESSIONID=B053D613E27F77C01F5142CDC9813C38'
-H 'Content-Length:0'
-H 'Authorization:Basic OjIyMDgzMzE4MDU1NDMxNzM5NjQ=' 
-d '' 
 

2024-03-26 15:56:06 INFO 	RESPONSE:
STATUS_CODE: [31m404[0m
DATA: [31mResponding with error, status code: 404 (Not Found).
Details: jetbrains.buildServer.server.rest.errors.NotFoundException: No build type nor template is found by id 'YVKCZRsoJht'.
Could not find the entity requested. Check the reference is correct and the user has permissions to access the entity.[0m
2024-03-26 15:56:06 INFO [32mpytest tests/backend/test_create_build_conf.py::TestBuildConfCreate::test_positive_creating_build_conf[By agent manager] (teardown)[0m
curl - X GET 'http://localhost:8111/authenticationTest.html?csrf' 
-H 'User-Agent:python-requests/2.31.0'
-H 'Accept-Encoding:gzip, deflate'
-H 'Accept:application/json'
-H 'Connection:keep-alive'
-H 'Content-Type:application/json'
-H 'X-TC-CSRF-Token:ab464863-767f-4d39-af89-f539280c81fd'
-H 'Cookie:TCSESSIONID=B053D613E27F77C01F5142CDC9813C38'
-H 'Authorization:Basic OjIyMDgzMzE4MDU1NDMxNzM5NjQ=' 
-d '' 
 

2024-03-26 15:56:06 INFO [32mpytest tests/backend/test_create_build_conf.py::TestBuildConfCreate::test_positive_creating_build_conf[By agent manager] (teardown)[0m
curl - X DELETE 'http://localhost:8111/app/rest/projects/id:CCnCQqCzWZW' 
-H 'User-Agent:python-requests/2.31.0'
-H 'Accept-Encoding:gzip, deflate'
-H 'Accept:application/json'
-H 'Connection:keep-alive'
-H 'Content-Type:application/json'
-H 'X-TC-CSRF-Token:ab464863-767f-4d39-af89-f539280c81fd'
-H 'Cookie:TCSESSIONID=B053D613E27F77C01F5142CDC9813C38'
-H 'Content-Length:0'
-H 'Authorization:Basic OjIyMDgzMzE4MDU1NDMxNzM5NjQ=' 
-d '' 
 

2024-03-26 15:56:06 INFO 	RESPONSE:
STATUS_CODE: [31m404[0m
DATA: [31mResponding with error, status code: 404 (Not Found).
Details: jetbrains.buildServer.server.rest.errors.NotFoundException: No project found by locator 'count:1,id:CCnCQqCzWZW'. Project cannot be found by external id 'CCnCQqCzWZW'.
Could not find the entity requested. Check the reference is correct and the user has permissions to access the entity.[0m